source/_posts/2023-08-02-release-20238.markdown

```diff
@@ -536,6 +536,24 @@ superseded by three single actions `start`, `stop` and `return_to_base`.
 
 {% enddetails %}
 
+{% details "Google Nest" %}
+
+The legacy Works With Nest API is no longer supported. You must manually take
+action to use the SDM API by removing nest configuration from configuration.yaml
+and restarting Home Assistant, then add the Nest integration following the
+instructions. Please review the documentation for information about which
+devices are supported.
+
+Additionally, as a reminder, the legacy Works With Nest API is [shutting down](https://developers.nest.com/)
+in September 2023.
```

Comment:
Remove this:
```suggestion
```

Reply:
Are you suggestion this because of my other comment about not blaming nest? I think your suggestion of linking to the deprecation docs seemed helpful, so i'm ok to leave this and its verbatim from the breaking change PR.

Reply:
I don’t know. I think we could add it back in and then merge the PR.

Reply:
OK, its there now.